rococo-parachains/tests/purge_chain_works.rs

```diff
@@ -15,50 +15,63 @@
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
 use assert_cmd::cargo::cargo_bin;
-use std::{convert::TryInto, fs, path::PathBuf, process::Command, thread, time::Duration};
+use std::{convert::TryInto, process::Command, thread, time::Duration};
 
 mod common;
 
 #[test]
 #[cfg(unix)]
 fn purge_chain_works() {
```

Comment:
Would be really nice to have these tests part of the `/tests/` "parachain". But this can be done in a later pr.